src/queries/useAuthorities/useAuthorities.js

```diff
@@ -103,7 +105,15 @@ const useAuthorities = ({
     .map(([filterName, filterValues]) => {
       const filterData = filterConfig.find(filter => filter.name === filterName);
 
-      return filterData.parse(filterValues);
+      let finalFilterValues = filterValues;
+
+      if (filterName === facetsTypes.SUBJECT_HEADINGS) {
```

Comment:
I think we can remove this mapping here if in `<MultiSelectionFacet>` in `options` we add `label` property
If `option.label` exists - then we can use it to display facet options, otherwise use `option.id`.
And in `getSubjectHeadingsFacetOptions` we can format options:
```
{
  id: value.id,
  label: subjectHeadingsName,
}
```

Reply:
Actually, I don't think it'll work. This mapping is needed because for this facet in `filterValues` we have facets' names (i.e. `['Medical Subject Headings']`) but backend expects facets' codes (i.e. `(subjectHeadings==("c"))`), so we have to change name to corresponding code before sending the request.
But I still added `label` property, thanks!